api/types/ui_config.go

```diff
@@ -113,6 +127,16 @@ func (c *UIConfigV1) SetSubKind(sk string) {
 	c.SubKind = sk
 }
 
+// GetShowResources will returns which resources should be shown in the unified resources UI
+func (c *UIConfigV1) GetShowResources() constants.ShowResources {
+	return c.Spec.ShowResources
+}
+
+// SetShowResources will set which resources should be shown in the unified resources UI
+func (c *UIConfigV1) SetShowResources(value constants.ShowResources) {
+	c.Spec.ShowResources = value
+}
```

Comment:
Is this being used?
I didn't find a reference to it.

Reply:
This method or the value? The method I added just because I've seen the get/set for the other values in ui/other configs.

Reply:
Method.
I think we can remove it, but no big deal

Reply:
Removed 👍